lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -66,24 +66,39 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                 description: """
                 Defines the features that will have to be supported by decoder
                 to decode video encoded with this element.
+                It sets an upper limit of compression and can ovverride other options
+                (e.g setting max_b_frames to 2 and profile to baseline, no b_frames will be present)
```

Comment:
OK, here's what I had in mind. Please take a look at it and check if it makes sense 
```suggestion
                Sets a limit on the features that the encoder will use to ones supported in provided H264 profile.
                Said features will have to be supported by the decoder in order to decode the resulting video.
                It may override other, more specific options affecting compression (e.g setting `max_b_frames` to 2
                and profile to `:baseline`, will have no effect and no B-frames will be present).
```